### ISSUE TITLE
A11Y bug: Errors disappear in subscription & contactforms

### ISSUE THREAD

**Issue by digitala11y** [Automattic/jetpack#14155]
Hello,
In jetpack subscription & contact form modules the error messages appear only for few seconds. The focus moves to the field that has the error & when user tabs out the error messages disappear. As a screen reader user I am unable to see the error messages using my arrow keys & this is a failure of WCAG SC 2.2.1 timing adjustable. 

The error messages must be available all time until the user successfully completes the form. Is there a hook or a temporary fix to achieve this until this is adopted into jetpack?

#### What happened instead

**Screenshots**

**Comment by jeherve** [comment]
Thanks for the report. Are you referring to the error messages that are displayed when you do not fill in all mandatory fields, for example?

If so, those errors are handled by your browser itself, but there is definitely some things we could do to improve things, notably with `aria-live`.

**Comment by digitala11y** [comment]
Hello,
Yes I was referring to the errors when required fields are not filled. The errors disappear which is not supposed to happen. As far as I see the aria-live is working properly & focus is also moving to the field that has the error. If we can keep the errors until the user fixes them & re-submit the form then it is compliant with WCAG 2.0

Here is an example of the same
http://pauljadam.com/demos/aria-describedby-validation.html
